yazi-fm/src/notify/notify.rs

```diff
@@ -0,0 +1,54 @@
+use std::rc::Rc;
+
+use ratatui::{buffer::Buffer, layout::{Constraint, Layout, Offset, Rect}, style::{Style, Stylize}, widgets::{Block, BorderType, Paragraph, Widget}};
+use yazi_core::notify::{Level, Message};
+
+use crate::{widgets::Clear, Ctx};
+
+pub(crate) struct Notify<'a> {
+	cx: &'a Ctx,
+}
+
+impl<'a> Notify<'a> {
+	pub(crate) fn new(cx: &'a Ctx) -> Self { Self { cx } }
+
+	fn chunks(area: Rect, messages: &[Message]) -> Rc<[Rect]> {
+		let chunks = Layout::horizontal([Constraint::Percentage(100), Constraint::Min(40)]).split(area);
```

Comment:
You may also write this as:

```
let chunks = Layout::horizontal([Constraint::Fill(1), Constraint::Length(40)]).split(area);
```

Reply:
Awesome, will apply this suggestion in the next commit, thanks!